core/SymbolRef.h

```diff
@@ -9,7 +9,11 @@ class Symbol;
 class GlobalState;
 struct SymbolDataDebugCheck {
     const GlobalState &gs;
-    const unsigned int symbolCountAtCreation;
+    const unsigned int classAndModulesAtCreation;
```

Comment:
wow, I won't be surpried if this has perf impact on debug builds.

Reply:
I can change this to use the total count.

Reply:
Fixed.